doc/source/user_guide/indexing.rst

```diff
@@ -933,6 +933,20 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+``iloc`` distinguishes between different types of boolean indexing.  If the type of boolean indexing is ``Series``,
```

Comment:
``iloc`` supports two kinds of boolean indexing.

Reply:
If the indexer is a boolean ``Series``, ...

Reply:
Done.